web/themes/custom/cloudy/templates/field/field--field-published-on.html.twig

```diff
@@ -1,7 +1,12 @@
-{% set icon %}
-  <i class="fas fa-calendar"></i>
-{% endset %}
-{% include "field.html.twig" with {
-  'margin': 2,
-  'icon': icon
+{% embed "field.html.twig" with {
+  'margin': '2'
 } %}
+  {% block field %}
+    {% set field_classes = field_classes|merge(['align-items-baseline']) %}
+    {{ parent() }}
+  {% endblock %}
+  {% block label %}
+    {% set title_classes = title_classes|merge(['mb-0', 'small']) %}
```

Comment:
I like how the label looks a little smaller than the date for this field. Good call. I wonder if we could repeat that pattern in our defaults 🤔